src/validators/date-time-after-or-equal.js

```diff
@@ -0,0 +1,48 @@
+import always from 'ramda/src/always';
+import moment from 'moment';
+
+const NOW = 'now';
+const defaultMessage = '<%= propertyName %> must be greater than or equal to <%= ruleParams[1] %>.';
+
+// Use object (it will be passed as reference at index.js) to represent the message,
+// because we want to change it dynamically based on the offset parameter
+// based on the offset parameters.
+const message = {};
+
+const validate = (val, ruleObj) => {
+  if (!val) {
+    return true;
+  }
+
+  let dateTime;
+  const dateTimeInputFormat = ruleObj.params[0];
+  const dateTimeInput = moment(val, dateTimeInputFormat);
+  let offset = Number(ruleObj.params[2]);
+  const unit = ruleObj.params[3] || 'days';
```

Comment:
since it's dateTimeAfter, should we use `seconds` instead of `days`?

Reply:
Yeah, I agree. I've update the diff.